app/models/cloud_object_store_container.rb

```diff
@@ -15,4 +15,5 @@ class CloudObjectStoreContainer < ApplicationRecord
   alias_attribute :name, :key
 
   supports_not :delete, :reason => N_("Delete operation is not supported.")
+  supports_not :cloud_object_store_container_clear
```

Comment:
Nah, I just asked @miha-plesko to drop the reason. This is most likely not displayed in the UI at all and only adds a burden to i18n to translate messages that are not used.

Actually I'm inclined to remove all `supports_not` reason that are on a base class